recipe/meta.yaml

```diff
@@ -1,12 +1,9 @@
 {% set name = "petsc4py" %}
 {% set version = "3.21.1" %}  # TODO: remove Cython pinning for petsc4py-3.20.0
 {% set sha256 = "ea8c6afb16541167d39f87d5fcad98c32d856fe8a2173504ef2a31c16647d53d" %}
-{% set build = 0 %}
+{% set build = 1 %}
 
 {% set version_xy = version.rsplit('.', 1)[0] %}
-{% if scalar == "real" %}
-{% set build = build + 100 %}
-{% endif %}
```

Comment:
While I agree that we don't need it, it will be a problem to remove it now, because build 1 will be considered lower priority than the previous real build, which was 100.

Instead, if we're removing this, let's add the 100 to the base build number. It will then go back to zero on the next version bump.

Reply:
+1 from me